### PR TITLE
WRP-6257: Layout: Added screenshot tests for `grow` prop of `Cell`

### PR DESCRIPTION
**Pull request by hoeunryuLG**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
A follow-up PR to add qa-sampler and screenshot tests for `grow` prop in `Cell` (enactjs/enact#3128)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- ~~Added qa-sampler for `Layout`~~
- Added screenshot tests

~~qa-sampler~~
~~- One Column and Nested three Rows with a total of 9 Cells to control them at once.~~
~~- Three Cells in a Layout to control them individually.~~

~~In principle, this should be added to enact, but since enact does not have qa-sampler, it is added to sandstone instead.~~

screenshot tests

* Row
  - Simply set the `grow` prop to check if each cell occupies the same area.
  - Set `grow` and `size={1000} to make sure it fills the remaining area of the container.
  - by setting `grow` and `size={1500}`
     - Check that it does not shrink below the specified size
     - When `wrap` is activated, cells are listed in two lines and check if the last cell fills up the container.
    
* Column
  - Simply set the `grow` prop to check if each cell occupies the same area.
  - Set `grow` and `size={100}` to make sure it fills the remaining area of the container.
  - Set `grow` and `size={500}` to make sure it doesn't shrink below the
      specified size.

* Layout
  - by setting `grow` and `size={1500}`
     - Check that it does not shrink below the specified size.
     - When `wrap` is activated, check if cells are listed in two lines, and the last cell fills up the container.
 
### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
enactjs/enact#3128 should be merged first.

### Links
[//]: # (Related issues, references)
WRP-6257
enactjs/enact#3128

### Comments
Enact-DCO-1.0-Signed-off-by: Hoeun Ryu <hoeun.ryu@lge.com>

### CODE REVIEW

File: samples/sampler/stories/qa/Layout.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import {boolean, select} from '@enact/storybook-utils/addons/controls';
+import Layout, {Cell, Column, Row} from '@enact/ui/Layout';
+
+export default {
+	title: 'Sandstone/Layout',
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Layout.js:5]: I'm not sure about adding this to sandstone qa-sampler. 🤔
How about adding `grow` prop to the existing enact ui/Layout sampler?

File: tests/screenshot/apps/components/Layout.js [hunk 2]
```diff
@@ -41,7 +41,25 @@ const rowTestCases = [
...
+	</Layout>,
+
+	<Row>
+		<Cell className={css.red} grow>Layout Head</Cell>
+		<Cell className={css.blue} grow>Layout Content</Cell>
+		<Cell className={css.green} grow>Layout Tail</Cell>
+	</Row>,
+
+	<Row>
+		<Cell className={css.red} grow size={1000}>Layout Head</Cell>
```
Comment (MikyungKim) [inline tests/screenshot/apps/components/Layout.js:53]: The above Row result and this Row result look the same. Is this intended? Or should we change the size..?

File: tests/screenshot/apps/components/Layout.js [hunk 3]
```diff
@@ -76,6 +94,24 @@ const columnTestCases = [
...
+	</Column>,
+
+	<Column style={{height: "500px"}}>
+		<Cell className={css.red} grow>Column header6</Cell>
+		<Cell className={css.blue} grow>Column Content6</Cell>
+		<Cell className={css.green} grow>Column footer6</Cell>
+	</Column>,
+
+	<Column style={{height: "500px"}}>
+		<Cell className={css.red} grow size={100}>Column header7</Cell>
```
Comment (MikyungKim) [inline tests/screenshot/apps/components/Layout.js:106]: Likewise, the previous one and this test look similar.

File: samples/sampler/stories/qa/Layout.js (hunk 1 above)
Comment (hoeunryuLG) [reply samples/sampler/stories/qa/Layout.js:5]: I thought sampler and qa-sampler had different purposes.
The sampler had a showcase for UI components, and the qa-sampler had various test cases for UI components. But Enact doesn't have a qa-sampler, so I put this code in Sandstone's qa-sampler. But I will if you want to place this code into Enact's sampler.

File: tests/screenshot/apps/components/Layout.js (hunk 2 above)
Comment (hoeunryuLG) [reply tests/screenshot/apps/components/Layout.js:53]: This is intended. The result of both is the same. However, I want to test that when the sum of the sizes of three Cells is smaller than the size of the container, the three Cells are arranged side by side and fill the remaining space of the container.

File: tests/screenshot/apps/components/Layout.js (hunk 3 above)
Comment (hoeunryuLG) [reply tests/screenshot/apps/components/Layout.js:106]: Same as the intent of the test above.

File: samples/sampler/stories/qa/Layout.js (hunk 1 above)
Comment (MikyungKim) [reply samples/sampler/stories/qa/Layout.js:5]: If we could cover the test with Screenshot tests, I think that is enough.
Let's not add this and see if you can add the `grow` prop to the default Layout sampler from enact sampler.

**Comment by hoeunryuLG** [comment]
The PR has been updated to address reviewer's comments.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**